src/index.ts

```diff
@@ -86,7 +86,12 @@ function activateFileBrowser(
       | string
       | null
       | undefined;
-    gitHubBrowser.baseUrl = baseUrl || DEFAULT_GITHUB_BASE_URL;
+    const accessToken = settings.get('accessToken').composite as
+      | string
+      | null
+      | undefined;
+    drive.baseUrl = baseUrl || DEFAULT_GITHUB_BASE_URL;
+    drive.accessToken = accessToken;
```

Comment:
We should also probably add a warning to the Javascript console that users should be careful with this setting.